task_8_9_5.py

```diff
@@ -0,0 +1,29 @@
+from utils import print_intro
+
+print_intro()
+
+MONTH_DAYS = {'january': '31',
+              'february': '28 or 29',
+              'march': '31',
+              'april': '30',
+              'may': '31',
+              'june': '30',
+              'july': '31',
+              'august': '31',
+              'september': '30',
+              'october': '31',
+              'november': '30',
+              'december': '31'}
+
+
+def print_days_in_month(month, days_in_month):
+    # Check if the correct month name has been entered
+    massage = f'Number of days in {month}: {days_in_month}' if days_in_month is not None \
+        else 'Invalid month name.'
+    print(massage)
```

Comment:
now it will be hard to test this function. make return here and print returned value outside.